cs/v1/CS_v1.0.0.yaml

```diff
@@ -1273,19 +1300,20 @@ components:
       title: Other Transport
       type: object
       description: Other Transport modes.
+      additionalProperties: false
       required:
         - modeOfTransport
       properties:
         modeOfTransport:
           type: string
+          description: "The mode of transport as defined by DCSA. The allowed values for the Other Transport mode are:\r\n- `RAIL` (When the transport mode is Rail)\r\n- `TRUCK` (When the transport mode is Truck)\r\n- `RAIL_TRUCK` (When rail and truck are expected to be the mode of transport in a leg of a proposed routing)\r\n- `BARGE_TRUCK`(When barge and truck are expected to be the mode of transport in a leg of a proposed routing)\r\n- `BARGE_RAIL`(When barge and rail are expected to be the mode of transport in a leg of a proposed routing)\r\n- `MULTIMODAL`(When mode of transport is not really defined or unknown at this stage)"
```

Comment:
Please format this text correctly and not use "Stoplight" version of inserting `\n`